docs/docs/aleph_zero/storage.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Storage
+
+slug: /aleph_zero/storage
+---
+
+## FeeTier
+
+```rust
+pub struct FeeTier {
+    pub fee: Percentage,
+    pub tick_spacing: u16,
+}
+```
+|Name|Type|Description|
+|-|-|-|
+|fee|Percentage|Percentage of the fee collected upon every swap in the pool|
+|tick_spacing|u16|The spacing between usable ticks|
+
+## PoolKey
+
+```rust
+pub struct PoolKey {
+    pub token_x: AccountId,
+    pub token_y: AccountId,
+    pub fee_tier: FeeTier,
+}
+```
+|Name|Type|Description|
+|-|-|-|
+|token_x|AccountId|address of token_x|
+|token_y|AccountId|address of token_y|
+|fee_tier|FeeTier|FeeTier associated with pool|
+
+## Pool
+
+```rust
+pub struct Pool {
+    pub liquidity: Liquidity,
+    pub sqrt_price: SqrtPrice,
+    pub current_tick_index: i32, // nearest tick below the current sqrt_price
```

Comment:
There should not be comments in state fields. If something needs elaboration should be done in the description